scripts/irods/test/test_auth.py

```diff
@@ -745,3 +745,126 @@ def test_iinit_with_pam_password_and_ssl(self):
         # Make sure that the authentication succeeded.
         assert_command('ils', 'STDOUT', f'/{self.zone}/home/{self.pam_user}', env=self.env)
 
+
+@unittest.skipIf(test.settings.USE_SSL, 'These tests set up SSL so if SSL is already set up, this should be skipped.')
+class test_pam_password(resource_suite.ResourceBase, unittest.TestCase):
```

Comment:
Do we need to inherit from `ResourceBase`?

Reply:
Mmmm... probably not. :) I believe I was being especially lazy when I wrote this so this can probably be pared down a lot. I'll look into it.

Reply:
Done